entity-types/ext-calico/dashboard.stg.json

```diff
@@ -27,7 +27,7 @@
                     [
                       {
                         "accountId": 0,
-                        "query": "SELECT latest(felix_cluster_num_hosts) AS 'Number of Calico Hosts' FROM Metric WHERE instrumentation.name = 'remote-write' AND prometheus_server IN ( '{{tags.prometheus_server}}') AND instance IN ( '{{tags.instance}}' ) AND entity.guid = '{{entity.id}}' facet prometheus_server"
+                        "query": "SELECT latest(felix_cluster_num_hosts) AS 'Number of Calico Hosts' FROM Metric WHERE instrumentation.name = 'remote-write' AND prometheus_server IN ( '{{tags.prometheus_server}}') AND instance IN ( '{{tags.instance}}' ) FACET prometheus_server"
```

Comment:
I removed `entity.guid = '{{entity.id}}'` because I noticed that the entity guid where clause was already added to the queries for us